app/views/instructor/courses/show.html.erb

```diff
@@ -41,7 +41,7 @@ $(function() {
 
 	<ul class="sections">
 		<% current_course.sections.rank(:row_order).each do |section| %>
-			<li data-update-url="<%= instructor_section_path %>">
+			<li data-update-url="<%= instructor_section_path(section) %>">
```

Comment:
The `instructor_section_path` needs the section for updating.  This populates the `params[:id]` and it's why:

```
Section.find(params[:id])
```

Works in the update method in the controller.